.pre-commit-config.yaml

```diff
@@ -23,9 +23,43 @@ repos:
           - '--args=--only=terraform_standard_module_structure'
           - '--args=--only=terraform_workspace_remote'
       - id: terraform_validate
+      - id: terraform_tfsec
```

Comment:
the variable changes are ok, but lets revert any changes on the pre-commit config